spec/operations/beta/project_board_operations_spec.rb

```diff
@@ -57,7 +57,8 @@
     describe 'when the project does not exist' do
       let(:user)    { create :user }
       let(:project) { create :project, users: [user] }
-      let(:result)  { Beta::ProjectBoardOperations::Read.call(nil, user) }
+
+      subject(:result) { Beta::ProjectBoardOperations::Read.call(user, nil) }
 
```

Comment:
I remember we had a broken case in the next PR we tried to handle that. Do we have a new test to cover that case to ensure it's working now?